src/styles/_table.scss

```diff
@@ -90,6 +90,10 @@ div.c-table {
         flex: 1 1 auto;
     }
 
+    &.is-stale {
+        border: 1px solid cyan;
```

Comment:
As mentioned above, let's move the `1px solid cyan` into a constant and include it in all the css styles.

Reply:
For @rukmini-bose and @charlesh88